### PR TITLE
test: add set cookie with all values test

### PR DESCRIPTION
**Pull request by kblok**
No code changes. I'm just adding a test to validate that we can send a valid cookie with all the available properties to the browser.

### CODE REVIEW

**Review by OrKoN — COMMENTED** [review]
Thanks for the test. I think not all cookie attributes work cross-browser. Perhaps we need to adjust the expectations base on the browser?

**Comment by kblok** [comment]
It looks stuck right @OrKoN? or Am I missing something to trigger the builds?

File: test/src/cookies.spec.ts [hunk 1]
```diff
@@ -311,6 +311,46 @@ describe('Cookie specs', () => {
         ]
       );
     });
+    it('should set cookie with all available properties', async () => {
+      const {page, server} = await getTestState();
```
Comment (kblok) [inline test/src/cookies.spec.ts:315]:
```suggestion
      const {page, server} = await getTestState();

```

**Comment by kblok** [comment]
@OrKoN it seems the build requires your approval to run?

**Comment by kblok** [comment]
> @kblok could you please run `npm run format:expectations` and commit changes?

done!

**Review by OrKoN — APPROVED** [review]

**Pull request merged**